proto/src/main/proto/Rss.proto

```diff
@@ -567,3 +569,17 @@ message ReassignServersReponse{
   bool needReassign = 2;
   string msg = 3;
 }
+
+message RssReassignFaultyShuffleServerRequest{
+  int32 shuffleId  = 1;
```

Comment:
Should we use int as shuffle id? Maybe we should use this for Tez, too. @zhengchenyu @lifeSo Could you take a look?

Reply:
In tez, it is ok to use int as shuffle id. Here is the shuffle id calc:


```
private static final int SHUFFLE_ID_MAGIC = 1000;

  public static int computeShuffleId(int tezDagID, int upVertexId, int downVertexId) {
    int shuffleId = tezDagID * (SHUFFLE_ID_MAGIC * SHUFFLE_ID_MAGIC)  + upVertexId * SHUFFLE_ID_MAGIC + downVertexId;
    LOG.info("Compute Shuffle Id:{}, up vertex id:{}, down vertex id:{}", shuffleId, upVertexId, downVertexId);
    return shuffleId;
  }
```

Reply:
Use `int` as `shuffleId` following the definitions of other messages.

Reply:
Yes